### PR TITLE
Addition of Idea Cards and dropdown menu area.

### PR DESCRIPTION
**Pull request by mdflynn**
## What is the change?

This adds the placeholder for the idea cards as well as the menu area for the dropdown page. 

## Is this a fix or a feature? 

This is a feature. It adds one of the last pieces of design for the mobile site. 

## Where should the reviewer start?

The reviewer should start at line 14 in the HTML and line 184 in the CSS file. 

## How should this be tested?

Open the index.html file in your terminal. Then open the chrome dev tools and change it to a mobile view. Then see the design updates.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,11 +11,20 @@
...
+              <span><img class="dropdown-icon" src="./assets/menu-close.svg" height="40" width="40"></i><span>
+            </div>
+            <div>
+              <h1 class="dropdown-ideabox">IdeaBox</h1>
+            </div>
+          </div>
+          <p class="filter-idea-text">Filter Starred Ideas</p>
+          <button class="starred-ideas-button">Show Starred Ideas</button>
+        </div>
+        <div class="opaque">stuff</div>
```
Comment (Estaffieri) [inline index.html:27]: Should "stuff" still be there? We can remove that in the next push.

**Review by Estaffieri — APPROVED** [review]
Really solid work on those idea cards and the overlay with the dropdown menu! This looks good and I think once we all merge our changes we need to check our naming conventions and make sure that they are semantic. Nice Job! 🤓⭐️

File: index.html (hunk 1 above)
Comment (mdflynn) [reply index.html:27]: Great catch! I've removed it from my code.

**Review by DennClark — COMMENTED** [review]
Looking great in the idea card section!

**Pull request merged**